src/main/io/beeper.h

```diff
@@ -31,6 +31,7 @@
 
 typedef enum {
     // IMPORTANT: these are in priority order, 0 = Highest
+    // Also, the order of the elements should be preserved for backwards compatibility with the configurator.
```

Comment:
I am not sure that the above comment is actually still correct. After all, we do have `uint8_t priority` in `beeperTableEntry_t`.

Reply:
@mikeller I've been looking at the code... if I'm not wrong, the MSP command only sends/reads a 32bit mask, that is verified against the `beeper.h` enum. So If I'm not wrong, we must maintain here the order. The text, priority, etc. of the beeperTableEntry_t is not used (at least not in the MSP/Configurator part). Maybe the plan was to add it to MSP later?